quartz/plugins/transformers/description.ts

```diff
@@ -19,19 +21,42 @@ export const Description: QuartzTransformerPlugin<Partial<Options> | undefined>
       return [
         () => {
           return async (tree: HTMLRoot, file) => {
-            const frontMatterDescription = file.data.frontmatter?.description
-            const text = escapeHTML(toString(tree))
+            let frontMatterDescription = file.data.frontmatter?.description
+            let text = escapeHTML(toString(tree))
+
+            if (opts.replaceExternalLinks) {
+              frontMatterDescription = frontMatterDescription?.replace(
+                /(https?:\/\/)?(?<domain>[\da-z\.-]+)\.([a-z\.]{2,6})(:\d+)?([\/\w\.-]*)/g,
+                "$<domain>",
+              )
+              text = text.replace(
+                /(https?:\/\/)?(?<domain>[\da-z\.-]+)\.([a-z\.]{2,6})(:\d+)?([\/\w\.-]*)/g,
+                "$<domain>",
+              )
```

Comment:
Can we make a regex for this similar to how we do regex in ofm?

Reply:
Thanks for the feedback!

I have updated the regex code to be more similar to the way in ofm.